.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Code owners
+
+# Reviewers for any change in the repository
+* @paolomorandini @mjquinn-google @bdw-g @shevek-google
+
+# Reviewers for Java changes
+/dumper-integration-tests/ @cyulysses-corp @haselbach
```

Comment:
Does this path actually exist? I see `/client/` in the repo but not this.

Reply:
This exists in the `dev` branch